colin/core/result.py

```diff
@@ -97,6 +99,44 @@ def save_json_to_file(self, file):
                   fp=file,
                   indent=4)
 
+    @property
+    def xunit(self):
+        """
+        Get the xunit representation of results
+
+        :return: str
+        """
+
+        top = Element("testsuites")
+
+        testsuite = SubElement(top, "testsuite")
+        top.append(testsuite)
+
+        for r in self.results:
+            testcase = SubElement(testsuite, "testcase", {
+                'name': r.check_name,
+                'status': "PASSED" if r.ok else "FAILED", # Can't use PASSED or FAILED global variables because their values are PASS and FAIL respectively.
+                'url': r.reference_url
+            })
+            testsuite.append(testcase)
+            logs = SubElement(testcase, "logs")
+            for log in r.logs:
+                log = SubElement(logs, "log", {
+                    'message': log,
+                    'result': 'INFO',
+                    'waiver_authorization': 'Not Waivable'
+                })
+                logs.append(log)
+
+        rough_string = ElementTree.toString(top, 'utf-8')
```

Comment:
it is `tostring` and not `toString` :-)

BTW `tostring` is a function in the module xml.etree.elementtree
https://docs.python.org/3.6/library/xml.etree.elementtree.html#xml.etree.ElementTree.tostring

And here the class is used xml.etree.ElementTree.ElementTree which does not have such method.
https://docs.python.org/3.6/library/xml.etree.elementtree.html#xml.etree.ElementTree.ElementTree